subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -54,6 +54,14 @@ abstract class InstrumentationTestsTask @Inject constructor(
     @InputFile
     val testApkOnTargetCommit: RegularFileProperty = objects.fileProperty()
 
+    @Optional
```

Comment:
https://github.com/gradle/gradle/issues/2016
be aware that `@Optional` + `@InputFile` is a dangerous combo

Reply:
I did it like other properties. For instance for APK.

Reply:
But thank you for pointing the issue